contracts/0.4.24/Lido.sol

```diff
@@ -902,14 +905,12 @@ contract Lido is StETHPermit, AragonApp {
         require(recipients.length == modulesFees.length, "WRONG_RECIPIENTS_INPUT");
 
         if (totalFee > 0) {
-            uint256 shares2mint = _totalRewards
-                .mul(totalFee)
-                .mul(_getTotalShares())
+            uint256 shares2mint =
+                _totalRewards.mul(totalFee).mul(_getTotalShares())
                 .div(
-                    _getTotalPooledEther()
-                        .mul(precisionPoints)
-                        .sub(_totalRewards.mul(totalFee))
-            );
+                    (_getTotalPooledEther().mul(precisionPoints))
+                    .sub(_totalRewards.mul(totalFee))
+                );
```

Comment:
Why this formatting change? The result is much less readable.

Reply:
That change pertains to a former base branch, `fix/typos-base-precision` which has already been merged to #482. This PR includes change only to StakingRouter and respective tests.